website/src/components/Roadmap.tsx

```diff
@@ -0,0 +1,78 @@
+import { Container } from "./Container";
+
+const Roadmap = () => {
+  return (
+    <Container className="">
+      <div className="py-32">
+        <h2 className="text-4xl mb-16">Our Roadmap</h2>
+        <div className="flex flex-col items-center space-y-8 md:space-y-0 md:items-start md:flex-row md:justify-between">
```

Comment:
@brentbagapuro thanks for the contribution!

I don't think it is necessary for this page since it only shows info with little interaction. But at some point we have to migrate to using chakra-ui.

For any future pull requests, prefer chakra-ui instead of raw html if possible ❤.

I cannot judge the content so I will wait for other reviewers from the team. But it looks good to me.

Reply:
Thank you for you review @AbdBarho ! Appreciate it.

I understand. Was only basing on current layout of the homepage as it's mostly Tailwind styling as I did here. I also consulted with @fozziethebeat about my approach to use Tailwind UI when asking to assign the issue to me.

Nevertheless, I'll make sure future PRs prioritize chakra-ui for building the UI.

Thank you again! ❤

Reply:
I answered in the issue a bit too quickly.  @AbdBarho is correct, Chakra-UI components are preferred over raw tailwind whenever possible but as a first draft it's not a big deal.  We can switch things out later

Reply:
Gotcha. I'll take note of that. Thanks!